mon-pix/tests/integration/components/navbar-desktop-header-test.js

```diff
@@ -109,6 +109,49 @@ describe('Integration | Component | navbar-desktop-header', function() {
     });
   });
 
+  context('when user comes from external platform', function() {
+    beforeEach(async function() {
+      this.owner.register('service:session', Service.extend({
+        isAuthenticated: false,
+        data: {
+          externalUser: 'externalUserToken',
+        },
+      }));
+      setBreakpoint('desktop');
+      await render(hbs`<NavbarDesktopHeader/>`);
+    });
+
+    it('should be rendered', function() {
+      // then
+      expect(find('.navbar-desktop-header__container')).to.exist;
+    });
+
+    it('should display the Pix logo', function() {
+      // then
+      expect(find('.navbar-desktop-header-logo')).to.exist;
+      expect(find('.pix-logo')).to.exist;
+    });
+
+    it('should not display the navigation menu', function() {
+      // then
+      expect(find('.navbar-desktop-header-container__menu')).to.not.exist;
+    });
+
+    it('should not display link to inscription page', function() {
```

Comment:
```suggestion
    it('should not display link to signup page', function() {
```
Inscription n'a pas ce sens en anglais
https://www.wordreference.com/definition/inscription

Reply:
Done.